src/client/QXmppPubSubManager.cpp

```diff
@@ -20,6 +21,8 @@
 
 using namespace QXmpp::Private;
 
+const char *PUBSUB_CURRENT_ITEM = "current";
+
```

Comment:
```suggestion
```

Reply:
Removed by 4711fe94c376e60744fb05c9e71563a958072db2

Reply:
please do it correctly with the enum in this commit already (instead of adding code that is removed in the next commit) :)